mirage/config.js

```diff
@@ -29,8 +29,22 @@ function routes() {
       is_sso: false,
       is_sso_enforced: false
     }
+  });
+
+  this.get('/api/v2/partnerclients', (schema) => {
+    return schema.partnerClients.all();
+  })
+
+  this.get('/api/v2/partnerclients/:id', (schema, FakeRequest) => {
+    return schema.partnerClients.findBy({
+      id: FakeRequest.queryParams.id
+    });
   })
 
+  this.get('/api/v2/partnerclients/:id/plan', () => {
+    return this.db.partnerclientPlans[0];
+  });
```

Comment:
Can you help me to understand what's the advantage of keeping these here rather than defining in respective test cases whenever required? I feel that would be more flexible

Reply:
1. All the endpoints will be organized in the config script. And also, if there is a requirement to get a model instance, calling create/createList would handle it. 
2. Complicated queries could be moved inside the test case itself but for simple and generic one can be maintained inside the config script.